### PR TITLE
chore(cms): add Open Robot Kit page

### PR DESCRIPTION
**Pull request by greatislander**

### CODE REVIEW

**Comment by sbates-idrc** [comment]
When I first looked at the Open Robot Kit page, and thought that the "Who We Are" section was the start of the site footer, rather than content that was specific to the Open Robot Kit. We could perhaps do the styling differently for this section, or adjust the wording to make it clearer that it's for the Open Robot Kit.

**Comment by sbates-idrc** [comment]
I have ordered the robot iterations in the https://github.com/sbates-idrc/c2lc-coding-environment/tree/open-robot-kit/open_robot_kit directory by adding number prefixes to the iteration directories

**Comment by sbates-idrc** [comment]
Dates of Tetra TMU project:

- Initial project: February 2021 - June 2021
- Continued development and integration: July 2021 - mid 2022
  - Liam continued on with Daniel; I’m not sure when this phase officially ended

@chosww and @michelled Do either of you have a record of the dates for the Tetra TMU project?

File: src/open-robot-kit.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - name: Inclusive Design Research Centre
+      acronym: IDRC
+      content: IDRC is an international community of open source developers,
+        designers, researchers, educators and co-designers who work together to
+        proactively ensure that emerging technology and practices are designed
+        inclusively.
+      link: https://idrc.ocadu.ca
+    - name: Tetra TMU
+      content: Tetra TMU is a student chapter under the Tetra Society of North America
+        operating at Ryerson University. We focus on bridging students and
```
Comment (sbates-idrc) [inline src/open-robot-kit.md:28]: "Ryerson" used here instead of "TMU"

**Comment by chosww** [comment]
> 

I checked text messages between Liam and I, and seems like our first meeting was on July 11th, 2022 and the last meeting was in mid October of 2022.

**Comment by chosww** [comment]
Initial project time seemed correct, and after that, I continued for another school year with new Tetra TMU students to complete one of the prototype (the one Liam's group made). Which ended end of school year of 2022, so beginning of May. After that, Liam and I continued from July to October of 2022.

**Comment by greatislander** [comment]
Hi all, I've expanded the page a bit and added the handbooks for each team. I think we could maybe add the prototype video of all three teams' robots, and possibly links to @sbates-idrc's iterations on the Art Bot. What do you think? https://deploy-preview-822--weavly.netlify.app/open-robot-kit/

**Comment by chosww** [comment]
Thanks @greatislander for the work, showing each team's work looks great, but I wonder if we could possibly use an overlay to display the PDF doc or open the PDF on a new tab? When each team's overview is clicked, we direct users to the PDF, and there's no way to get back to the page other than pressing the back button from the browser, which doesn't feel natural. If we want to keep the flow the way it is now, then maybe we could add aria-label to inform users that they will be direct to a browser PDF.

**Pull request merged**